### ISSUE TITLE
Unexpected error on mef_eline logs after redeploying some EVCs

### ISSUE THREAD

**Issue by italovalcy** [kytos-ng/mef_eline#542]
Hi,

After asking for the redeployment of some EVCs, each one of them at a time spaced by 5s sleep time, we eventually see the following error on the logs:

```
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[1276, 1276]] was already available in00:00:00:00:00:xx:xx:xx:30
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[1276, 1276]] was already available in00:00:00:00:00:xx:xx:xx:4
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[2519, 2519]] was already available in00:00:00:00:00:xx:xx:xx:1
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[2519, 2519]] was already available in00:00:00:00:00:xx:xx:xx:1
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[2489, 2489]] was already available in00:00:00:00:00:xx:xx:xx:2
2024-10-21 04:04:48,205 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[2489, 2489]] was already available in00:00:00:00:00:xx:xx:xx:2
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[2769, 2769]] was already available in00:00:00:00:00:xx:xx:xx:1
2024-10-21 04:04:48,238 - INFO [uvicorn.access] [httptools_impl.py:506:send] (MainThread) 10.0.0.2/fulano:0 - "PATCH /api/kytos/mef_eline/v2/evc/ed919d5a34b446/redeploy HTTP/1.1" 202
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[2769, 2769]] was already available in00:00:00:00:00:xx:xx:xx:1
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[1982, 1982]] was already available in00:00:00:00:00:xx:xx:xx:2
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[1982, 1982]] was already available in00:00:00:00:00:xx:xx:xx:1
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:52:make_vlans_available] (AnyIO worker thread) Tags [[1846, 1846]] was already available in00:00:00:00:00:xx:xx:xx:2
2024-10-21 04:04:48,206 - ERROR [kytos.napps.kytos/mef_eline] [path.py:55:make_vlans_available] (AnyIO worker thread) Tags [[1846, 1846]] was already available in00:00:00:00:00:xx:xx:xx:2

```

Since later on I saw some inconsistencies on the logs (not necessarily related to this), I think it is worthy reporting and checking why we saw this error (Maybe a race-condition??).

**Comment by viniarck** [comment]
Indeed it can be a race cond, basically, all the `log.error` with "was already available" were cases that were added that wasn't expected to be hit in normal circumstances.

**Comment by viniarck** [comment]
This will be fixed on this PR that's will land https://github.com/kytos-ng/topology/pull/233. 

Other than that I also found https://github.com/kytos-ng/topology/issues/229, which also has another PR in-flight to land soon. 

Finally, I didn't found any other race in this part. I also ran a custom stress scripts for 48 hours and no conflicting s vlans were observed when dealing with redeploys and some link downs, each iteration was being checked for conflicting svlans in the same link.
